profiles/base.yaml

```diff
@@ -504,6 +506,28 @@ workorder_max_items: 3
 workorder_recipes:
 workorder_cash_on_hand:
 
+#CARAVAN SETTINGS
+# container holding caravan contracts for ;trade
+trade_contract_container: backpack
+# number of copper to carry on-hand while running ;trade
+caravan_coins_on_hand: 9555
+# Have you purchased a 750plat interior from the trade minister?
+caravan_interior: false
+# skills and cooldowns to train in ;trade.  See https://github.com/rpherbig/dr-scripts/wiki/Trader-Tutorials for valid skills and what they do
+caravan_training_skills:
```

Comment:
❌ This needs an entry in `base-empty.yaml`, something like `caravan_training_skills: {}`